src/EnumDamageCauseCode.src.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "EnumDamageCauseCode",
+  "uri": "http://schema4i.org/EnumDamageCauseCode",
+  "description": "Code for coding the damage types according to BiPRO.",
```

Comment:
Remove BiPRO. Even if parts of the codes come from official standards the list in general might now or later contain other codes that are not part of the referred standards